content/applications/sales/point_of_sale/overview/getting_started.rst

```diff
@@ -2,148 +2,105 @@
 Getting started
 ===============
 
-When working with a Point of Sale application, employees want a
-simple, and user-friendly solution. A solution that works online or
-offline and with any device.
+**Point of Sale** offers employees a simple and user-friendly solution that works online or offline
+with any device.
 
-A Point of Sale system is a fully integrated application that allows any transaction, 
-automatically registers product moves in your stock, and gives you
-real-time statistics and consolidations across all shops.
+A point of sale system is a fully integrated application that allows any transaction, automatically
+registers product moves in your stock, and gives you real-time statistics and consolidations across
+all shops.
 
 Configuration
 =============
 
-Make products available in the PoS
-----------------------------------
+Make products available
+-----------------------
 
-To make products available for sale, go to :menuselection:`Point of
-Sale --> Products --> Products` and open a product. In the *Sales*
-tab, enable *Available in Point of Sale*.
+To make products available for sale, go to :menuselection:`Products --> Products`, and open a
+product. In the :guilabel:`Point of Sale` tab, enable :guilabel:`Available in POS`.
 
-.. image:: getting_started/getting_started_01.png
-    :align: center
-    :alt: Product form. Making the product available in PoS.
-
-.. note::
-   You can also define if the product has to be weighted or not.
+.. image:: getting_started/pos-available.png
+   :align: center
+   :alt: Making a product available in PoS.
 
 Configure your payment methods
 ------------------------------
 
-To add a payment method, you first need to create it. Go to
-:menuselection:`Point of Sale --> Configuration --> Payment Methods`
-and click on create. Then, you can name your payment method and set it
-up.
+To add a payment method, you first need to create it. Go to :menuselection:`Configuration -->
+Payment Methods -->Create`. Name your payment method and select the **payment terminal** or check
+:guilabel:`Cash` for cash payments.
 
-.. image:: getting_started/getting_started_02.png
-    :align: center
-    :alt: Creating a new payment method for a Point of Sale.
+.. image:: getting_started/payment-method.png
+   :align: center
+   :alt: Creating a new payment method for a Point of Sale.
 
-.. note::
-   Don’t forget your credentials for methods using a payment terminal.
+Now, you can select the payment method in your POS settings. To do so, go to
+:menuselection:`Point of Sale --> Configuration --> Point of Sale` and select a POS. Then, go to
```

Comment:
```suggestion
:menuselection:`Point of Sale --> Configuration --> Point of Sale` and select a POS. Then, go to
```
You added "Point of Sale" in the menuselection here, but line 18 you just entered Products --> Products, I suggest you either remove it, or add it to line 18, for consistency.

Reply:
You're right, I'm taking it off